src/python/pants/backend/go/util_rules/first_party_pkg_test.py

```diff
@@ -167,9 +178,10 @@ def test_invalid_package(rule_runner) -> None:
     )
     assert maybe_info.info is None
     assert maybe_info.exit_code == 1
-    assert maybe_info.stderr == "bad.go:1:1: expected 'package', found invalid\n"
+    assert "bad.go:1:1: expected 'package', found invalid\n" in maybe_info.stderr
 
 
+@pytest.mark.xfail(reason="cgo is ignored")
```

Comment:
I've made the analyzer ignore cgo for now. I plan to address cgo analysis in a separate PR.